presto-hive/src/main/java/com/facebook/presto/hive/PrestoS3SignerType.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive;
+
+import static java.util.Objects.requireNonNull;
+
+public enum PrestoS3SignerType
```

Comment:
This violates normal enum naming conventions (but is fine in this case), so we should add
```java
@SuppressWarnings("EnumeratedConstantNamingConvention")